src/security/cert_generators/file/certificate_db.cc

```diff
@@ -107,12 +105,12 @@ Ssl::Locker::Locker(Lock &aLock, const char *aFileName, int aLineNo):
 
 Ssl::Locker::~Locker()
 {
+    (void)lineNo; // lineNo is unused but some find it helpful in debugging
     if (weLocked)
         lock.unlock();
 }
 
-Ssl::CertificateDb::Row::Row()
-    :   width(cnlNumber)
+Ssl::CertificateDb::Row::Row() : width(cnlNumber)
```

Comment:
If this is a manual formatting-only change, please undo it.

Reply:
It's not manual